app-officeext/ooofbtools/ooofbtools-2.38.ebuild

```diff
@@ -0,0 +1,40 @@
+# Copyright 1999-2015 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+# $Id$
+
+EAPI=5
+
+MY_PN="OOoFBTools"
+
+OFFICE_EXTENSIONS=(
+	"${MY_PN}.oxt"
+)
+
+inherit office-ext-r1
+
+DESCRIPTION="Extension for converting and processing eBooks in FictionBook2 format with validator"
+HOMEPAGE="https://sourceforge.net/projects/fbtools/"
+SRC_URI="mirror://sourceforge/fbtools/files/release/${MY_PN}-${PV}.zip"
+
+LICENSE="GPL-3"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+
+DEPEND=""
+RDEPEND="dev-libs/libxml2"
+
+S="${WORKDIR}/${MY_PN}-${PV}"
+
+OFFICE_EXTENSIONS_LOCATION="${S}"
+
+src_prepare() {
+	# Remove Windows cruft
+	pushd "${WORKDIR}/${MY_PN}.oxt" 2>/dev/null
```

Comment:
`pushd` and `popd` can fail in the same way `cd` can fail, so there are `|| die`s missing